test/manifest_test.rb

```diff
@@ -0,0 +1,24 @@
+require "webpacker_test"
+
+class ManifestTest < Minitest::Test
+  def test_file_path
+    file_path = File.join(File.dirname(__FILE__), "test_app/public/webpack", "manifest.json").to_s
+    assert_equal WebpackerLite::Manifest.file_path.to_s, file_path
+  end
+
+  def test_lookup_exception
+    manifest_path = File.join(File.dirname(__FILE__), "test_app/public/webpack", "manifest.json").to_s
+    asset_file = "calendar.js"
+
+    error = assert_raises WebpackerLite::FileLoader::NotFoundError do
+      WebpackerLite::Manifest.lookup(asset_file)
+    end
+
+    assert_equal error.message, "Can't find #{asset_file} in #{manifest_path}. Is webpack still compiling?"
+  end
+
+  def test_lookup_success
+    asset_file = "bootstrap.js"
```

Comment:
Usually, a variable in these tests is a literal that the result of the tested method is compared to. In this case, it's the literal of the tested method's argument. Breaks expectations.